profiles/commands.py

```diff
@@ -0,0 +1,16 @@
+from flask_script import Command
+
+
+class ClearCommand(Command):
+    def __init__(self, *repositories):
```

Comment:
Can we add a `CanBeCleared` type? We might not just want to clear repositories (thinking caches).